### PR TITLE
style : alert. & button CSS updated

### PR DESCRIPTION
**Pull request by Bibin-Antony**
minor changes made and updated on buttons

### CODE REVIEW

File: Container/CSS/button.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    background: var(--tertiary-color);
+}
+.danger-btn{
+    color: var(--text-color);
+    background: var(--danger-alert);
+}
+.warning-btn{
+    color: var(--text-color);
+    background: var(--warning-alert);
+}
```
Comment (laxmikanthkonthum) [inline Container/CSS/button.css]: since the text-color is repeating we can try writing it under a common class

**Review by laxmikanthkonthum — APPROVED** [review]
added a comment and rest looks good to me. Also if possible pls try to attach a screenshot of your output from next time

**Review by maitrakhatri — APPROVED** [review]
looks good to me

**Comment by Bibin-Antony** [comment]
image

File: Container/CSS/button.css (hunk 1 above)
Comment (Bibin-Antony) [reply Container/CSS/button.css]: yes, ill figure out how to do that and do it.

**Comment by Bibin-Antony** [comment]
changes made 
- comments added
- put things into a common class to reduce repetaion. 
- reference image added

**Pull request merged**